pkg/objstore/inmem.go

```diff
@@ -24,14 +24,18 @@ type InMemBucket struct {
 	mtx     sync.RWMutex
 	objects map[string][]byte
 	attrs   map[string]ObjectAttributes
+
+	downloadedCount     map[string]uint
```

Comment:
I would vote for just using MetricBucket wrapper instead. You can just use count metric (not needed per file) right?

The first reason is that we already have an implementation for tracking this. Secondly, it's because it's not testing bucket (by name), it's in mem bucket. While it's recommended for testing usage, let's keep it production-grade if we can (: 

WDYT?